### DISCUSSION TITLE
Alpha on startup has the wrong colours and seems to require an action before correction

### DISCUSSION THREAD

**Discussion by dlynz** [NvChad/NvChad#2471] [Q&A] [answered]
On nvim startup, the landing page looks like this with a very different colour scheme.
![image](https://github.com/NvChad/NvChad/assets/100419289/0f5aba1c-e874-4393-a100-22a6eb7cc773)
As compared to when I do an action like opening telescope and it turns back into the correct colours for my theme which is catppuccin (syntax highlighting problem?)
![image](https://github.com/NvChad/NvChad/assets/100419289/d86577d1-3fbd-46a1-9bcf-0ac51eb8ed01)
Any ideas why this is happening? I'm assuming it's with the order of loading highlighting and such, but I can't figure out how to fix it.

**Comment by siduck** [comment, marked as answer]
https://nvchad.com/docs/config/theming#extended_integrations

**Reply by dlynz** [reply]
Hi I seem to be misunderstanding or something.
Heres my current chadrc and alpha plugin code am I missing something?
```---@type ChadrcConfig
local M = {}
M.ui = {
    theme = 'catppuccin',
    theme_toggle = { "catppuccin", "ayu_light" },

    transparency = true,
}
M.ui.extended_integrations = {'alpha'}
M.plugins = 'custom.plugins'
return M
```

```
local plugins = {
    {
        "goolord/alpha-nvim",
        event = "VimEnter",
        config = function ()
            dofile(vim.g.base46_cache .. "alpha")
            require'alpha'.setup(require'alpha.themes.dashboard'.config)
        end
    },
}

return plugins
```

**Reply by siduck** [reply]
@dlyynn i think our alpha hlgroups were for our old alpha config, can you try "syntax" instead of alpha in dofile code?

**Reply by dlynz** [reply]
ahh that did it thank you very much :)

**Reply by siduck** [reply]
btw @dlyynn we have our dashboard module, you can use that as your alpha setup looks simple!
